contracts/rust/diff-test/src/main.rs

```diff
@@ -503,9 +512,17 @@ fn main() {
 
                 ledger.elapse_with_block();
 
-                let (pi, proof) = ledger.gen_state_proof();
+                let (pi, proof, stt) = ledger.gen_state_proof();
                 new_states.push(pi.into());
                 proofs.push(proof.into());
+
+                let parsed_stake_state = ParsedStakeState {
+                    threshold: field_to_u256(stt.threshold),
+                    bls_key_comm: field_to_u256(stt.stake_table_bls_key_comm),
+                    schnorr_key_comm: field_to_u256(stt.stake_table_schnorr_key_comm),
+                    amount_comm: field_to_u256(stt.stake_table_amount_comm),
+                };
+                stake_stakes.push(parsed_stake_state.into());
```

Comment:
Looks like `stake_stakes` vector is not returned. Why is that?

Reply:
this will be removed in my future commit of mine

Reply:
removed in [af51261](https://github.com/EspressoSystems/espresso-sequencer/pull/1986/commits/af5126122179c8c58999c32b3a084b1bd8e4a2c0)